sycl/doc/extensions/supported/sycl_ext_oneapi_bfloat16.asciidoc

```diff
@@ -96,52 +86,45 @@ enum class aspect {
 }
 ----
 
-If a SYCL device has the `ext_oneapi_bfloat16` aspect, then it natively
-supports conversion of values of `float` type to `bfloat16` and back.
-
-If the device doesn't have the aspect, objects of `bfloat16` class must not be
-used in the device code.
-
-**NOTE**: The `ext_oneapi_bfloat16` aspect is not yet supported.  The
-`bfloat16` class is currently supported only on Xe HP GPU and Nvidia GPUs with Compute Capability >= SM80.
+This extension is an optional kernel feature as described in section 5.7 of the SYCL 2020 spec, with the associated aspect `ext_oneapi_bfloat16`. Applications can query whether the device has this aspect to determine if it supports `bfloat16` conversions.
```

Comment:
This sentence is also outdated (assuming there is fall-back support).

I'd propose moving the description of this aspect to the bottom of the "Specification" section in a new sub-section named "Deprecated features".  That section can say something like:

> This extension adds a new aspect named `ext_oneapi_bfloat16`, but usage of this aspect is deprecated.  It used to indicate whether a device supports bfloat16, but all devices are now required to support bfloat16 when an implementation supports this extension.  Therefore, this aspect now returns `true` for all devices.

In order to avoid losing this information:

> The bfloat16 class is currently supported in hardware on Xe HP GPUs and Nvidia GPUs with Compute Capability >= SM80. On other devices it is emulated in software.

You can add a new NOTE to the end of the "Overview" section that says something like:

> The `bfloat16` type is supported on all devices.  DPC++ currently supports this type natively on Intel Xe HP GPUs and Nvidia GPUs with Compute Capability >= SM80.  On other devices it is emulated in software.

Reply:
Yes, that is a good idea. Done.